CMakeLists.txt

```diff
@@ -494,7 +502,7 @@ if(BUILD_BINDINGS_PYTHON AND NOT BUILD_SHARED_LIBS)
   set(BUILD_BINDINGS_PYTHON OFF)
 endif()
 if(BUILD_BINDINGS_PYTHON)
-  set(BUILD_BINDINGS_PYTHON_VERSION ${PYTHON_VERSION_MAJOR})
+  set(BUILD_BINDINGS_PYTHON_VERSION 3.6...<3.11)
```

Comment:
I believe it should be Python_VERSION instead of Python_VERSION_MAJOR. If we are using python version 3.10, for example, Python_VERSION would be "3.10" and Python_VERSION_MAJOR would be "3".

I just submitted a fix.